proxy.go

```diff
@@ -121,13 +121,46 @@ func setupLogger(logLevel string) error {
 	return nil
 }
 
+func printAgentLogs(sock string) error {
+	// Don't return an error if nothing has been provided.
+	// This flag is optional.
+	if sock == "" {
+		return nil
+	}
+
+	agentLogsAddr, err := unixAddr(sock)
```

Comment:
Thinking about security here... can we validate the permissions on this socket and reject it if it's read/writeable by "other"?

Reply:
Sure, I will check for that.